dalm/pipelines/README.md

```diff
@@ -0,0 +1,16 @@
+## Reading comprehension Pipeline
+
+This is an example of how to string together an end to end reaading-comprehension dataset generator and training regiment in one go
+
+It should be noted that this not the last say, if you the user think there's a better of stringing these together, the components
+are atomic enough for you to string them in other ways that you may deem more appropriate
+
+```bash
+python dalm/pipelines/reading_comprehension_pipeline.py --model_name HuggingFaceH4/zephyr-7b-alpha \
+    --input input.csv --csv_column text \
+	--output_dataset_name combined \
+    --general_spm_path tokenizers/general.spm \
```

Comment:
I think this argument merits some documentation that describes how to download or create this SPM tokenizer.

Also, could this parameter be made optional?  If not specified, it could just download a default tokenizer and allow users to customize this if they had a need?  This should reduce the cognitive load on users trying to run this command.

Reply:
From the CLI help:

> path to the general tokenizer (needed for regex based generation)

It's not clear to me if this is a location where a tokenizer will be saved, or if its the location where the code expects to find an existing tokenizer.

Reply:
Also if the default `--general_spm_path` value works, any reason to highlight it in the example command?